### ISSUE TITLE
Language conditioned baseline

### ISSUE THREAD

**Issue by Aopolin-Lv** [Stardust-hyx/Bridge-Torch#1]
Do you have plan to reimplement the baseline of language condition model, such as RT-1?

**Comment by Stardust-hyx** [comment]
Yes. However, I am currently busy with other stuff. I will carry out relevant implementations after November.




何羽鑫
***@***.***

stardust-hyx.github.io

---Original---
From: "Qi ***@***.***&gt;
Date: Wed, Oct 25, 2023 10:47 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [Stardust-hyx/Bridge-Torch] Language conditioned baseline (Issue #1)




 
Do you have plan to reimplement the baseline of language condition model, such as RT-1?
